app/controllers/concerns/sign_in/authentication.rb

```diff
@@ -15,10 +17,11 @@ module Authentication
     def authenticate
       @access_token = authenticate_access_token
       @current_user = load_user
+      sign_in_logger.authenticate_access_token_stats(:success)
```

Comment:
I don't think we'll want a statsd for authenticate, since it'll be used for all routes. Doesn't seem like it gives us much useful info to have a metric for arbitrary authenticated route successes